oeo.omn

```diff
@@ -102,13 +109,38 @@ ObjectProperty: conforms_to
     
 ObjectProperty: contains
 
+
     
 ObjectProperty: covers
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A relation that holds between a publication/model calculation/model and the thing it covers."@en
     
     
+ObjectProperty: covers_demand_sector
+
+    SubPropertyOf: 
+        covers_sector
+    
+    
+ObjectProperty: covers_energy_sector
+
+    SubPropertyOf: 
+        covers_sector
```

Comment:
we should delete "covers_energy_sector" and "covers_demand_sector" though because the sector discussion isn't decided yet.

Reply:
I agree.